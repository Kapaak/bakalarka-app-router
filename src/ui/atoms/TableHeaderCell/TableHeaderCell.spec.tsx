import { render } from "@testing-library/react";

import { TableHeaderCell } from "./TableHeaderCell";

describe("TableHeaderCell", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<TableHeaderCell title="" icon={null} />);

    expect(baseElement).toBeTruthy();
    expect(baseElement).toMatchSnapshot();
  });
});
