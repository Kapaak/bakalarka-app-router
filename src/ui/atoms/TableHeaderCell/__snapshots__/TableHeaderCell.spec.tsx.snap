// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHeaderCell should render successfully 1`] = `
<body>
  <div>
    <div
      class="mr-5 flex flex-col gap-1 text-start"
    >
      <div />
      <h2 />
    </div>
  </div>
</body>
`;
